packaging/rpm/microshift.spec

```diff
@@ -269,4 +273,4 @@ mv /usr/lib/systemd/system/microshift-containerized.service /usr/lib/systemd/sys
 - add missing BuildRequires on systemd and policycoreutils
 
```

Comment:
Let's add a packaging entry log here.

Please try it with make srpm it doesn't complain about the entrylog.